src/node/desktop/src/ui/modal-dialog.ts

```diff
@@ -84,6 +85,11 @@ export abstract class ModalDialog<T> extends BrowserWindow {
     // show the window after loading everything
     this.show();
 
+    const showDevTools = getenv('RSTUDIO_DESKTOP_MODAL_DEVTOOLS').length !== 0;
+    if (showDevTools) {
+      this.webContents.openDevTools();
+    }
+
```

Comment:
You could skip backporting all the changes in this source file. They aren't actually a core part of the fix (but also relatively harmless).

Reply:
I've removed this file's changes